src/neo/Wallets/Wallet.cs

```diff
@@ -341,7 +341,7 @@ private Transaction MakeTransaction(StoreView snapshot, byte[] script, Signer[]
                     Version = 0,
                     Nonce = (uint)rand.Next(),
                     Script = script,
-                    ValidUntilBlock = snapshot.Height + Transaction.MaxValidUntilBlockIncrement,
+                    ValidUntilBlock = snapshot.Height + ProtocolSettings.Default.MaxValidUntilBlockIncrement,
```

Comment:
Not related to this PR, should we use the last known height instead of our height?

Reply:
Maybe the wallet should not allow the user to send transactions before the blocks are fully synchronized.